server/pulp/server/db/model/__init__.py

```diff
@@ -645,3 +645,60 @@ class CeleryBeatLock(Document):
 
     # For backward compatibility
     _ns = StringField(default='celery_beat_lock')
+
+
+class ContentCatalogEntry(Document):
+    """
+    A catalog of content that can be downloaded by the specified plugin.  Entries
+    are grouped by revision.  Each revision represents a platform event that
```

Comment:
The revision concept was replace with the concept that importers would add/update the catalog entries instead.  I just need to purge it from the docstrings.